Basic/Basic/DetailViewController.m

```diff
@@ -0,0 +1,20 @@
+//
+//  DetailViewController.m
+//  Basic
+//
+//  Created by Dima on 07.08.17.
+//  Copyright © 2017 Dima. All rights reserved.
+//
+
+#import "DetailViewController.h"
+
+@implementation DetailViewController
+
+- (void) viewDidLoad
+{
+    [self setTitle: self.employee.fullName];
```

Comment:
Access and set properties with dot notation. So change `[self setTitle: self.employee.fullName];` to `self.title = self.employee.fullName;`
Change it all over the project.